lib/src/google_fonts_family_with_variant.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/foundation.dart';
+import 'package:google_fonts/src/google_fonts_variant.dart';
+
+/// Represents a Google Fonts API variant in Flutter-specific types.
+class GoogleFontsFamilyWithVariant {
+  const GoogleFontsFamilyWithVariant({
+    @required this.family,
+    @required this.googleFontsVariant,
+  })  : assert(family != null),
+        assert(googleFontsVariant != null);
+
+  final String family;
+  final GoogleFontsVariant googleFontsVariant;
+
+  GoogleFontsFamilyWithVariant.fromApiFilenamePrefix(String apiFilename)
```

Comment:
I may be missing something, but I can't figure out where you actually use this constructor.

Reply:
You are right this is unused. I think I started out with it for symmetry with toApiFilenamePrefix, but it ended up not being needed